File: src/main/java/akostenko/aicars/plots/EmptyDriver.java
package akostenko.aicars.plots;

import akostenko.aicars.race.Driver;

public class EmptyDriver extends Driver {
    @Override
    public double accelerating() {
        return 0;
    }

    @Override
    public double breaking() {
        return 0;
    }

    @Override
    public double steering() {
        return 0;
    }

    @Override
    public String getName() {
        return "no name";
    }

    @Override
    public void update(double dTime) {

    }
}

File: src/main/java/akostenko/aicars/plots/EmptyDriver.kt
package akostenko.aicars.plots

import akostenko.aicars.race.Driver

class EmptyDriver : Driver() {
    override fun accelerating(): Double {
        return 0.0
    }

    override fun breaking(): Double {
        return 0.0
    }

    override fun steering(): Double {
        return 0.0
    }

    override val name: String
        get() = "no name"

    override fun update(dTime: Double) {

    }
}
